custom/import_xml.php

```diff
@@ -121,71 +121,77 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
   if ($olddata['squad'] && ! acl_check('squads', $olddata['squad']))
    die("You are not authorized to access this squad.");
 
-  newPatientData(
-   $olddata['id'],
-   $apatient['title'],
-   $apatient['fname'],
-   $apatient['lname'],
-   $apatient['mname'],
-   $apatient['sex'],
-   $apatient['dob'],
-   $apatient['street'],
-   $apatient['zip'],
-   $apatient['city'],
-   $apatient['state'],
-   $apatient['country'],
-   $apatient['ss'],
-   $apatient['occupation'],
-   $apatient['phone_home'],
-   $apatient['phone_biz'],
-   $apatient['phone_contact'],
-   $apatient['status'],
-   $apatient['contact_relationship'],
-   $apatient['referrer'],
-   $apatient['referrerID'],
-   $apatient['email'],
-   $apatient['language'],
-   $apatient['ethnoracial'],
-   $apatient['interpreter'],
-   $apatient['migrantseasonal'],
-   $apatient['family_size'],
-   $apatient['monthly_income'],
-   $apatient['homeless'],
-   fixDate($apatient['financial_review']),
-   $apatient['pubpid'],
-   $pid,
-   $olddata['providerID'],
-   $apatient['genericname1'],
```

Comment:
genericname, genericval stuff should be taken out.  That is not used.  More importantly, the actual table structure should be queried (select *) for what the fields are.  Matching can be done after that.